test/test-liquid.js

```diff
@@ -481,3 +481,44 @@ test('fastify.view with liquid engine template that does not exist errors correc
     })
   })
 })
+
+test('reply.view with liquid engine, should allow multiple root directories', t => {
+  t.plan(7)
+  const fastify = Fastify()
+  const { Liquid } = require('liquidjs')
+  const data = { text: 'text' }
+
+  const engine = new Liquid({
+    root: ['./', './templates']
+  })
+
+  fastify.register(require('../index'), {
+    engine: {
+      liquid: engine
+    }
+  })
+
+  fastify.get('/', (req, reply) => {
+    reply.view('./templates/index.liquid', data)
```

Comment:
```suggestion
    reply.view('index.liquid', data)
```
If you provide only the template name you actually test your feature.
(As it is now your test pass in the main branch ^^).

Reply:
Please feel free to see the updated code in next push

Reply:
Thank you :)
I will take a look in a few minutes :)